### DISCUSSION TITLE
I am stucked at the the group_expr parameter in the Relation API , please help with examples

### DISCUSSION THREAD

**Discussion by duyvanda** [duckdb/duckdb#8164] [Q&A] [answered]
I am trying the DuckDBPyRelation Python API, but when I come to group_expr parameter like:

value_counts(self: [duckdb.DuckDBPyRelation](https://duckdb.org/docs/archive/0.5.1/api/python/reference/#duckdb.DuckDBPyRelation), value_counts_aggr: str, group_expr: str = '')

or
sum(self: [duckdb.DuckDBPyRelation](https://duckdb.org/docs/archive/0.5.1/api/python/reference/#duckdb.DuckDBPyRelation), sum_aggr: str, group_expr: str = '')

They take group_expr as a parameter and I try all possible strings but nothing works:

dbr = db.sql("from df_dms")

dbr.sum(group_expr="stt, lat")

dbr.sum(group_expr="'stt', 'lat'")

dbr.sum(group_expr="['stt', 'lat']")

dbr.sum(group_expr="('stt', 'lat')")

and I have spent 2 hours on this.

Thanks for help

**Comment by Mause** [comment]
You need to pass a value for the sum_aggr parameter as well, have you tried that?

**Reply by duyvanda** [reply]
![image](https://github.com/duckdb/duckdb/assets/68499531/5fe97c28-052b-49aa-93a1-54cc28f1388e)

@Mause Thanks for replying, I want it to show the columns slsperid, checktype in the result as well. How can I archive that?

Thanks

**Reply by Tishj** [reply, marked as answer]
```py
sum = dbr.sum(...)
rel = dbr.query("dbr, "select slsperid, checktype, (select * from sum) from dbr")
rel.df()
```
I think that should work
